File: BackEnd/Api/WebApi/Models/Order.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Api.Services.Models;

namespace Api.WebApi.Models
{
    public class Order
    {
        public int OrderId { get; set; }
        public string PhoneNumber { get; set; }
        public string OrderName { get; set; }
        public int OrderStatus { get; set; }
        public DateTime OrderTime { get; set; }
        public string OrderPrice { get; set; }
        public int LocationId { get; set; }
        public List<Pizza> Pizzas { get; set; }
        public int PizzaId { get; set; }
    }
}

File: FrontEnd/src/app/models/order.ts
export class Order{
    public orderLocation: string;
    public orderIdNumber: string;
    public orderPizzas: string;
    public orderStatus: string;
    public orderName: string;
    public orderPrice: string;
    public orderPhoneNumber: string;
    public orderTime: string;

    constructor(orderLocation: string, orderIdNumber: string, orderPizzas: string, orderStatus: string,
    orderName: string, orderPrice: string, orderPhoneNumber: string, orderTime: string)
    {
        this.orderLocation = orderLocation;
        this.orderIdNumber = orderIdNumber;
        this.orderPizzas = orderPizzas;
        this.orderStatus = orderStatus;
        this.orderName = orderName;
        this.orderPrice = orderPrice;
        this.orderPhoneNumber = orderPhoneNumber;
        this.orderTime = orderTime;
    }
}